device.html

```diff
@@ -5,3 +5,10 @@
 ---
 
 <p>The Tamagotchi Architecture: Device</p>
+
+<p>The examination of the very original version of Tamagotchi,to Tamagotchi Plus to the most recent mobile app version reveals that all of the devices are designed and manufactured in the shape of an egg. This makes logical sense given three main reasons: first, “Tama” means an egg in the Japanese language while “gotchi” stands for watch. Then an egg pops up on the screen of the device and a Tama pet hatches from the egg after the owner activates the device. The last but not the least in terms of sensationalizing customers, the egg shape gives more of a smooth shape feeling compared to a rectangular or square shape. This makes the Tamagotchi device more appealing and convenient to carry in the hand.  Moreover, the variety of color and pattern design on the egg shape attracts a young generation of customers.
+In regard to the size of Tamagotchi, Tamagotchi mini comes in 1.2 x 0.5 x 1.5 inches. Though the size may vary in different versions, they all serve the same purpose as handheld portable devices. Moreover, all the devices are equipped with a LCD screen in the center of the device to serve as a media to present all the Tama pets activity and engagement.  The thing that varies is the screen evolves from black and white to color screen. For example, the resolution for the screen of Tamagotchi connexion 1 is 32 x 20. In addition all the devices have three buttons on the front bottom side, and they are named with the letters: A, B, C from left to right. Those buttons serve as tolls for the users to click on in order to interact with the pet and give it commands while the commands vary in different scenarios. For example, pressing A allows the users to navigate between different options, and pressing B allows you to choose the option to activate its command to the pet. On the back of a Tamagotchi, it is the battery case and a reset button, which customers can replace new batteries by removing the screws and reset the device. And each new device comes alone with a safety protection which is used to guarantee a brand new device is delivered to the customers.
+        	The last, some versions of Tamagotchi come with an infrared connector that allows users to connect to another device so that their Tama pets can interact with each other. This largely increases the variety of engagement for the customers to experience.
+</p>
+
+<img src="{{{'/build/img/front.jpg' | url}}">
```

Comment:
There's one extra `{` at the beginning of the image link. Other than that, good to go!